incubator/hnc/Makefile

```diff
@@ -143,11 +149,21 @@ deploy: docker-push kubectl manifests
 deploy-watch:
 	kubectl logs -n hnc-system --follow deployment/hnc-controller-manager manager
 
-# Installs cert-manager. See https://cert-manager.io/docs/installation/kubernetes/ for why we're not validating our YAML.
-deploy-cm:
-	@kubectl cluster-info
-	-kubectl create namespace cert-manager
-	kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml
```

Comment:
Probably mention removing the `deploy-cm` target in the commit message too?

Reply:
Done!